common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -462,6 +462,14 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     REPL_DUMPDIR_TTL("hive.repl.dumpdir.ttl", "7d",
         new TimeValidator(TimeUnit.DAYS),
         "TTL of dump dirs before cleanup."),
+    REPL_DUMP_METADATA_ONLY("hive.repl.dump.metadata.only", false,
+        "Indicates whether replication dump only metadata information or data + metadata."),
+    REPL_DUMP_INCLUDE_ACID_TABLES("hive.repl.dump.include.acid.tables", false,
+        "Indicates if repl dump should include information about ACID tables. It should be \n"
+            + "used in conjunction with 'hive.repl.dump.metadata.only' to enable copying of \n"
+            + "metadata for acid tables which do not require the corresponding transaction \n"
+            + "semantics to be applied on target. This can be removed when ACID table \n"
```

Comment:
This is user facing description, so last sentence can be rephrased as - "This configuration option will be dropped when ACID table replication is supported".

Reply:
will do